shtdlib.sh

```diff
@@ -193,7 +193,7 @@ function debug {
         if [ -e "${init_tty}" ] ; then
             color_echo yellow "${*:2}" > "${init_tty}"
         else
-            color_echo yellow "${*:2}"
+            color_echo yellow "${*:2}" >&2
```

Comment:
General comment... have you considered setting a file for stderr or even a new descriptor for our own verbose logging?

Reply:
@hrpatel